lib/color_guard/feature.rb

```diff
@@ -0,0 +1,69 @@
+require 'zlib'
+
+module ColorGuard
+  class Feature
+    attr_accessor :users, :percentage
+    attr_reader :name
+
+    def initialize(name, string = nil)
+      @name = name
+
+      if string
+        raw_percentage, raw_users = string.split("|")
```

Comment:
Its the serialization format. `new` is also serving to parse existing features.

This code was more-or-less copied from the Rollout gem where the entire class is private so I'm open to changing it, but that is the thought process